Makefile

```diff
@@ -40,7 +40,9 @@ unit-test:
 	poetry run pytest tests/unit
 
 e2e-test:
-	python parallel_run_e2e.py
+	poetry run pytest tests/e2e
+
+# python parallel_run_e2e.py
```

Comment:
Remove or comment this for future changes?

Reply:
removing it.